specification/api-tracing.md

```diff
@@ -386,6 +387,18 @@ Attributes SHOULD preserve the order in which they're set. Setting an attribute
 with the same key as an existing attribute SHOULD overwrite the existing
 attribute's value.
 
+Attribute values expressing a numerical value of zero or an empty string are
+considered meaningful and MUST be stored and passed to the exporter.
+Attribute values of `null` are considered to be not set and discarded as if
+`SetAttribute` was never called.
+If overwriting values is allowed, this results in clearing the previous value
```

Comment:
```suggestion
If changing/editing values is allowed, this results in clearing the previous value
```

Reply:
I would not use "overwriting is allowed" because in `SetAttribute` the overwriting of values is allowed. However, when a Span is terminated, calling again `SetAttribute` on an existing label must not change the attribute value.

Reply:
I took the wording over from L387 above.

Reply:
And by _allowed_ I rather meant to say _implemented_ since overwriting is a SHOULD requirement and not a MUST.
How about "If overwriting of attributes is implemented, attempting to set `null` results in clearing the previous value"?

Reply:
I suggest "supported", https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/9#discussion_r378887521